packages/pieces/community/ever-gauzy/src/index.ts

```diff
@@ -0,0 +1,62 @@
+import { createPiece, PieceAuth, OAuth2PropertyValue } from "@activepieces/pieces-framework";
+import { OAuth2GrantType, PieceCategory } from "@activepieces/shared";
+import { createEmployee, deleteEmployee, updateEmployee, getEmployee, listEmployees } from './lib/actions/employee';
+import { startTimer, stopTimer, getTimerStatus } from "./lib/actions/timer";
+import { newTimeTrack, newTask, newEmployee } from "./lib/triggers";
+import { createTask, deleteTask, updateTask, getTask, getListOfTasks } from "./lib/actions/tasks";
+import { createCustomApiCallAction } from '@activepieces/pieces-common';
+
+export const gauzyAuth = PieceAuth.OAuth2({
+  required: true,
+  grantType: OAuth2GrantType.AUTHORIZATION_CODE,
+  authUrl: 'https://api.gauzy.co/api/auth/auth0',
```

Comment:
@RolandM99 can this be configuration parameters or something? I.e. base URL for Gauzy API?